aiohttp/connector.py

```diff
@@ -470,25 +479,34 @@ def _get(self, key):
         del self._conns[key]
         return None
 
+    def _release_key_waiter(self, key, waiters):
+        if not waiters:
+            return False
+
+        waiter = waiters.popleft()
+        if not waiter.done():
+            waiter.set_result(None)
+
+        if not waiters:
+            del self._waiters[key]
+
+        return True
```

Comment:
If you don't do anything with the result, don't implement a result at all, it's like dead code. This is just a small remark

Reply:
Ah I see sry